Rakefile

```diff
@@ -15,17 +15,6 @@ rescue LoadError
   end
 end
 
-begin
```

Comment:
Why didn't we want cook style in rake? That just make it easier for people to run it locally before they push up a PR?

Reply:
I was following something that Tim did but you're right, I'll add it back in.

Reply:
The other thing is that on Windows, there are ALWAYS carriage returns at the end of the line so running the command line here always throws errors. OK I tweaked it in the rakefile to ignore the CRLF on Windows but only if you call "rake style" locally. The lint.yml file still runs all cops